res/values-ar/strings.xml

```diff
@@ -1326,7 +1326,7 @@ Commands:
     <string name="Allow_Recombine">السماح بإعادة التجميع</string>
     <string name="Defenders">المُدافِعون</string>
     <string name="Attackers">المُهاجِمون</string>
-    <string name="MEGA_SPLIT">64x قسمة كبير</string>
+    <string name="MEGA_SPLIT">قسمة كبير</string>
```

Comment:
Please do not remove that. It may not appear in the English string (blue), but "64x" remains in every other language.

Reply:
> Please do not remove that. It may not appear in the English string (blue), but "64x" remains in every other language.

exactly